cypress/cypress.config.js

```diff
@@ -0,0 +1,30 @@
+const { defineConfig } = require('cypress')
+const fs = require('fs')
+
+module.exports = defineConfig({
+    e2e: {
+        videosFolder: 'videos/',
+        videoUploadOnPasses: false,
+        supportFile: false,
+        specPattern: './**/*.cy.{js,jsx,ts,tsx}',
+        setupNodeEvents(on) {
+            const filesToDelete = []
+            on('after:spec', (spec, results) => {
+                if (results.stats.failures === 0 && results.video) {
+                    filesToDelete.push(results.video)
+                }
+            })
+            on('after:run', async () => {
+                filesToDelete.forEach((path) => {
+                    fs.unlink(path, (err) => {
+                        if (err) {
+                            console.error(err)
+                            return
+                        }
+                        console.log('Successfully deleted passing test video ' + path)
+                    })
+                })
+            })
```

Comment:
See the `videoUploadOnPasses` setting [here](https://docs.cypress.io/guides/references/configuration#Videos). Doesn't that effectively do the same?

We also have an account for cypress cloud, so we should register the project there, and set a project ID here. That opens up the possibility of running tests in parallel as well.

Reply:
I had some issues getting getting Cypress not outputting videos even with that property.

If we end up using Cypress we should definitely do that.